aikau/src/main/resources/alfresco/lists/Paginator.js

```diff
@@ -468,7 +521,7 @@ define(["dojo/_base/declare",
                }
                
                // Create the page labels, which for English will be along the lines of 1-25
-               if (this.compactMode === false)
+               if (this.compactMode === false && this.showPageSelector)
```

Comment:
No, because we had compactMode and can't remove it (backwards compatibility) and showPageSelector does something different.